user-guide/Advanced_Modules/Data_Aggregator/Data_Aggregator_settings.md

```diff
@@ -40,6 +40,24 @@ For example:
 > - Keep in mind that **if there is no firewall in place, anyone can use the web UI and the REST API**, as no authentication is required to use Data Aggregator.
 > - To go to the debug page, add `/debug` to the URL. For example, if the URL was defined as `http://127.0.0.1:5000`, you can access the debug page at `http://127.0.0.1:5000/debug`.
 
+## GQI DxM
+
+By default, queries are executed using CoreGateway and SLHelper.
+To execute the queries using the [GQI DxM](xref:GQI_DxM), the following setting must be enabled:
+
+```json
+{
+  "QueryExecutorOptions": {
+    "UseGQIDxM": true
+  }
+}
+```
+
+> [!NOTE]
```

Comment:
This note is probably important enough to be the first thing you see in this section?
Maybe even have a higher priority than just Note (e.g. Important/Warning)?